lib/govuk_index/elasticsearch_saver.rb

```diff
@@ -0,0 +1,32 @@
+module GovukIndex
+  class ElasticsearchSaver
+    TIMEOUT_SECONDS = 5.0
+
+    def save(presenter)
+      client.bulk(
+        index: index_name,
+        body: [
+          { index: presenter.identifier },
+          presenter.document
+        ]
+      )
+    end
+
+    def client(options = {})
```

Comment:
Should this be a private method? I can't see why it would want to share the client with other objects.